website_and_docs/content/blog/2022/python_locators_se4.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Locate Your Locators - Python Bindings Changes Upcoming"
+linkTitle: "Locate Your Locators Python Bindings Changes Upcoming"
+date: 2022-02-01
+tags: ["webdriver", "python"]
+categories: ["general"]
+author: Josh Grant ([@joshin4colours](https://twitter.com/joshin4colours))
+description: >
+  Updating Your Locators for Changes in the Pythom Bindings in Selenium 4
```

Comment:
Same as above
https://www.selenium.dev/documentation/about/style/#capitalization-of-titles

Reply:
Well, this is not a title, but I hope you get what I mean.